### ISSUE TITLE
Rename versions so they are sorted chronologically

### ISSUE THREAD

**Issue by jcwren** [DutchDevelop/BLLEDController#28]
It would be helpful to use the date format YYYY-MM-DD in the binary file names so that newer versions sort to the bottom.

**Comment by no-Legend** [comment]
👍

**Comment by ECKEROTH** [comment]
Yes please 👍🏻

**Comment by madnotna** [comment]
It would be nice if Dutch is following this:
https://en.wikipedia.org/wiki/Software_versioning

**Comment by webdeck** [comment]
Agreed that the version numbering is very confusing the way it sorts.

**Comment by vanowm** [comment]
More and more software adapting [CalVer](https://calver.org/), which makes total sense, it intuitive, sortable and it removes the need do any research if one needs to know when the version was released.

**Comment by vanowm** [comment]
I just realized, is version in this project - date backwards??? This is most insane versioning I've ever seen! On top of that installing "stable" version actually installs "experimental".
